edu.wpi.first.wpilib.plugins.java/src/main/resources/java-zip/ant/robotCommand

```diff
@@ -1,2 +1,2 @@
-env LD_LIBRARY_PATH=/usr/local/frc/rpath-lib/:/usr/local/frc/lib /usr/local/frc/bin/netconsole-host /usr/local/frc/JRE/bin/java -jar /home/lvuser/FRCUserProgram.jar
```

Comment:
Why not?  /usr/local/frc/lib is in /etc/ld.so.conf(.d), so it's not necessary to list in LD_LIBRARY_PATH.